tests/robot-tests/tests/admin_and_public/bau/publish_release.robot

```diff
@@ -100,7 +100,7 @@ Verify newly published release is on Find Statistics page
 Navigate to newly published release page
     [Tags]  HappyPath
     user clicks element   css:[data-testid="view-stats-ui-tests-publish-release-%{RUN_IDENTIFIER}"]
-    user waits until page contains heading  ${PUBLICATION_NAME}
+    user waits until page contains heading  ${PUBLICATION_NAME}  300
```

Comment:
Waiting for 5 minutes here seems extreme. Given that the publication should already be published I don't see why we'd need to wait this long. If it doesn't appear after a minute, I'd probably say it was never going to appear.

Reply:
I think the first time a release page compiles, this can take a long time. But I'll lower it a little.

Reply:
My general feeling here is I'd prefer to wait too long rather and definitely remove the timing issue when looking at intermittent failures